HW12/Bank_deposit.py

```diff
@@ -15,8 +19,9 @@ def register_client(self, name):
             "start_balance": None,
             "years": None
         }
-        print(f"Приветствуем нового клиента банка {name} "
-              f"с индивидуальным номером {self.client_id}")
+        if __name__ == "__main__":
+            print(f"Приветствуем нового клиента банка {name} "
+                  f"с индивидуальным номером {self.client_id}")
```

Comment:
I don't like this approach, it would be perfect to remove it from here and all other places in this module